quiz.py

```diff
@@ -0,0 +1,344 @@
+import os
+import random as r
+from datetime import datetime
+from time import sleep
+
+import matplotlib
+import numpy as np
+from PIL import Image, ImageDraw
+from matplotlib import patheffects
+from matplotlib import pyplot as plt
+from matplotlib.cbook import get_sample_data
+from matplotlib.offsetbox import (OffsetImage, AnnotationBbox)
+from telegram import Poll, ParseMode, Update
+from telegram.ext import CallbackContext
+from telegram.utils.helpers import mention_html
+
+from constants import group_ids
+from helpers.logger import logger
+from helpers.namer import get_nick
+from online import quiz_scraper
+
+quizzes = []
+cwd = os.getcwd()
+
+
+def send_quiz(context: CallbackContext) -> None:
+    """Sends 5 quizzes to target chat (12B for now). Also sets a timer for 24 hours for quiz expiry (using jobs)."""
+
+    global quizzes
+
+    right_now = datetime.now()  # returns: Datetime obj
+    if 'last_quiz' not in context.bot_data:
+        context.bot_data['last_quiz'] = right_now
+
+    diff = right_now - context.bot_data['last_quiz']
+    print(diff)
```

Comment:
Make print statement more informative; something like
`print(f"Last quiz was sent {diff.days} days ago.")`

Reply:
Uh okay will make that logging